htdocs/langs/fr_FR/admin.lang

```diff
@@ -1004,6 +1004,9 @@ Permission23001=Voir les travaux planifiés
 Permission23002=Créer/Modifier des travaux planifiées
 Permission23003=Effacer travail planifié
 Permission23004=Exécuter travail planifié
+Permission40001=Consulter les devises et leurs taux de change
```

Comment:
Can you add the new translation entries into the file in en_US file instead of fr_FR ?
Other language files, including french, will be automaticaly included into transifex platform for the translation team. Sync en_US to transifex is done once a week, and sync all translated languages to Dolibarr is done before beta and before each release.

Reply:
Hello,
I added the translations entries in the en_US file as requested, as well as fixed the stickler-ci continuous integration test error.